packages/core/contracts/financial-templates/long-short-pair/LongShortPairCreator.sol

```diff
@@ -123,15 +127,15 @@ contract LongShortPairCreator is Testable, Lockable {
         shortToken.addBurner(lspAddress);
         shortToken.resetOwner(lspAddress);
 
-        emit CreatedLongShortPair(lspAddress, msg.sender);
+        emit CreatedLongShortPair(lspAddress, msg.sender, address(longToken), address(shortToken));
 
```

Comment:
OZ comment on events

In `LongShortPairCreator.sol` there are no events emitted for the [creation of the `longToken`](https://github.com/UMAprotocol/protocol/blob/b508f536ddfd94a79f93f633142bdc868b73461c/packages/core/contracts/financial-templates/long-short-pair/LongShortPairCreator.sol#L84-L89) or [the `shortToken`](https://github.com/UMAprotocol/protocol/blob/b508f536ddfd94a79f93f633142bdc868b73461c/packages/core/contracts/financial-templates/long-short-pair/LongShortPairCreator.sol#L90-L95), nor are those addresses part of the [emitted `CreatedLongShortPair` event](https://github.com/UMAprotocol/protocol/blob/b508f536ddfd94a79f93f633142bdc868b73461c/packages/core/contracts/financial-templates/long-short-pair/LongShortPairCreator.sol#L126). Consider emitting events that contain the addresses of the new tokens to facilitate tracking.